### ISSUE TITLE
[Project creation] Categories

### ISSUE THREAD

**Issue by elievischel** [FreshRSS/FreshRSS#1989]
Create a project for :

Categories Management
- Feeds in multiple catégories
- Select and move multiple feeds at one time to another category
- Share a category amongst all the users

TAG management 
- add tag to feeds
- add tag to posts

**Comment by Alkarex** [comment]
Hello,
There is already a number of open issues regarding categories and tag. It would be good to link to them.
Yes for the tags, yes for moving several feeds at a time, but probably no for placing feeds in multiple categories (as discussed elsewhere, this would make the SQL requests significantly heavier, the UI more complicated, and the APIs as well as the compatible mobile clients are not compatible with the concept).

**Comment by elievischel** [comment]
@Alkarex Thank you for this, 

What about parent category ? 

If I can help in any way, it may be interesting to create projects for this git in order to split all the work and get contributors help :) in specific project.


it will give a pipeline for the different realease as well !! 

looking forward to your message, 
Best

**Comment by yllekz** [comment]
Hello, is it possible to please focus on allowing tagging feeds into multiple categories? Importing OPML from Inoreader (which allows feeds to be assigned to multiple categories) is a rather painful experience due to restricting it to one category only.

**Comment by Alkarex** [comment]
@yllekz Having the same feed under multiple categories is not something that is on the near-future roadmap. However, we could implement some workarounds to ease importing OPML from systems which support this approach. Could you share a short example of such an OPML?

**Comment by yllekz** [comment]
Absolutely. Here's an OPML file export containing subscriptions belonging to multiple categories:
https://1drv.ms/u/s!Arp9XZavoR5HhuNuMbTaZoKuFbOEsA?e=aUAjQw

**Comment by Alkarex** [comment]
Right, the feeds are kept under the first category under which they appear. I do not have a quick fix at the moment except moving them manually in the desired category

**Comment by Alkarex** [comment]
Slight improvement with https://github.com/FreshRSS/FreshRSS/pull/3286

**Comment by zombiehoffa** [comment]
This is really unfortunate that supporting multiple categories isn't on the near term road map, as it is a deal killer feature for me. Does not being on the near term road map mean don't expect it in six months or is it a nice way of saying it's probably not happening?

**Comment by NovaCyntax** [comment]
Wild.
